data/data-pipeline/data_pipeline/etl/sources/census_acs_2010/etl.py

```diff
@@ -50,7 +50,6 @@ def __init__(self):
         ]
 
         self.EMPLOYMENT_LESS_THAN_HS_IN_LABOR_FORCE = (
-            # TODO: FIX!!!!!!
             "B23006_005E"
```

Comment:
I wrote this scary todo a while ago; it should never have been committed. This used to be the wrong field. Now it's the right field, it's already fixed.